src/helpers/MdObjectHelper.ts

```diff
@@ -84,16 +89,13 @@ export const mdObjectToGeoPushpinBucketProps = (
     const locationBucket = findBucketByLocalIdentifier(contentBuckets, LOCATION);
     const segmentByBucket = findBucketByLocalIdentifier(contentBuckets, SEGMENT);
     const sizeBucket = findBucketByLocalIdentifier(contentBuckets, SIZE);
-    const tooltipTextBucket = findBucketByLocalIdentifier(contentBuckets, TOOLTIP_TEXT);
 
     const color: VisualizationInput.AttributeOrMeasure = colorBucket && colorBucket.items[0];
     const location: VisualizationInput.IAttribute =
         locationBucket && (locationBucket.items[0] as VisualizationObject.IVisualizationAttribute);
     const segmentBy: VisualizationInput.IAttribute =
         segmentByBucket && (segmentByBucket.items[0] as VisualizationObject.IVisualizationAttribute);
     const size: VisualizationInput.AttributeOrMeasure = sizeBucket && sizeBucket.items[0];
-    const tooltipText: VisualizationInput.IAttribute =
-        tooltipTextBucket && (tooltipTextBucket.items[0] as VisualizationObject.IVisualizationAttribute);
 
     // we'll use the default internal sorting value which
     // allow us to show the smaller pushpins on top of bigger pushpins
```

Comment:
these comment is not correct any more, right?

Reply:
removed